javascript/javascript1/week3/ItemRemoval.js

```diff
@@ -0,0 +1,11 @@
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+for ( let i = 0 ; i < names.length ; i++) {
+    
+    if (names[i].includes(nameToRemove)){
```

Comment:
actually this won't work. `includes` is an array method, and `names` is an array, but `names[i]` is a string! 
Hint: you don't need a for loop here, but you can use `indexOf`. Fix it.

Reply:
const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
const nameToRemove = 'Ahmad';


 const namesIndex = names.indexOf(nameToRemove) ;
 //console.log(namesIndex); For reference

 //console.log(names[namesIndex]);

    if (names[namesIndex] === nameToRemove)  {
        names.splice(namesIndex , 1);
    }

console.log(names);     // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']

Reply:
I have re-written the code as suggested